index.js

```diff
@@ -3,13 +3,20 @@ const BbPromise = require('bluebird');
 const jsdom = BbPromise.promisifyAll(require('jsdom'));
 const fs = BbPromise.promisifyAll(require('fs'));
 const streamBuffers = require('stream-buffers');
+const events = require('events');
```

Comment:
This can just be:

`const EventEmitter = require('events')`;

You can see in the node repl it is the same thing:

```
> require('events')
{ [Function: EventEmitter]
  EventEmitter: [Circular] // same,
  usingDomains: true,
  defaultMaxListeners: [Getter/Setter],
  init: [Function],
  listenerCount: [Function] }
```